CHANGES.md

```diff
@@ -1,3 +1,7 @@
+## Next
+
+ * Support GHC 8.6.1
```

Comment:
This changelog entry makes no sense whatsover; as you can see on

https://matrix.hackage.haskell.org/package/cassava

`cassava-0.5.0.0` as well as `cassava-0.5.1.0` obviously "support" GHC 8.6.1 already!

Reply:
Huh, are those running with `allow-newer`?

Reply:
No... what makes you think that? :-)

Reply:
I needed to `--allow-newer cassava:containers` to get my test-pilot project to build. I've just realised that that's because `doctest` and hence `ghc` is also in the build graph, so I guess the matrix builder's free to re-install `containers` where my set-up wasn't.